### DISCUSSION TITLE
LTSP discussions are here!

### DISCUSSION THREAD

**Discussion by alkisg** [ltsp/ltsp#159] [General]
LTSP applied and got accepted in the github discussions beta program, so [discussions](https://github.com/ltsp/ltsp/discussions) got enabled for the LTSP repository.

This means that the [LTSP community repository](https://github.com/ltsp/community) is now obsolete.
Its [issues](https://github.com/ltsp/community/issues) will be moved into discussions,
and its [wiki](https://github.com/ltsp/community/wiki) will be moved into the [LTSP repository wiki](https://github.com/ltsp/ltsp/wiki).

This will be done over the next few days.
Please do not file community issues anymore, but start ltsp discussions instead.

https://github.com/ltsp/ltsp/discussions

**Comment by alkisg** [comment]
Hi @MacMilanzi,

When you want to start a discussion, please go to https://github.com/ltsp/ltsp/discussions, click the `New discussion` button, write an appropriate title and your question, and finally press the `Start discussion` button.

This discussion that I started here was an announcement, it's not convenient to mix announcements and support topics in the same discussion, just start new ones.

**Comment by MacMilanzi** [comment]
@alkisg Thanks, will do that.
